src/test/objectstore/allocsim/ops_frag_reuse_check.cc

```diff
@@ -0,0 +1,173 @@
+#include "ops_parser.h"
+#include <vector>
+#include <string>
+#include <iostream>
+#include <boost/program_options/value_semantic.hpp>
+#include <boost/program_options/variables_map.hpp>
+#include <boost/program_options/parsers.hpp>
+
+using namespace std;
+using namespace ceph;
+namespace po = boost::program_options;
+
+void usage(po::options_description &desc) {
+  cout << desc << std::endl;
+}
+
+int op_comparison_by_object_date(const Op& lhs, const Op& rhs) {
+  if (*lhs.object < *rhs.object) return true;
```

Comment:
returning t/f from an int function?

Returning a bool is, of course, the first option, but you may want to consider <compare.h>, and
return strong_ordering